src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -172,6 +175,18 @@ class LinkView extends ViewFormBase<ILinkViewProps, ILinkViewState> {
         }
     };
 
+    private savePrompt = () => {
+        const confirmStore = this.props.confirmStore!;
```

Comment:
if !isNewLink call this.save() straightly? Now if creating a new link, it shows prompt which is unnecessary.